src/components/Map/GeoJSONTileLayer.js

```diff
@@ -252,20 +266,22 @@ class GeoJSONTileLayer extends TileLayer {
       tileLayer._tileOnLoad(tile, url);
     };
 
-    tile.request = { abort() {} };
     if (isCordova() && this.options.cordova) {
       const options = { headers: { Accept: 'application/json' } };
       fetchViaCordova(url, options)
         .then(r => r.text())
-        .then(responseText => loadGeoJSON(responseText))
+        .then(responseText => !tileRequestAborted && loadGeoJSON(responseText))
         .catch(error => tileLayer._tileOnError(tile, url));
     } else {
       const request = new XMLHttpRequest(); // eslint-disable-line no-param-reassign
       tile.request = request;
       request.open('GET', url, true);
       request.setRequestHeader('Accept', 'application/json');
+      request.addEventListener('abort', function load() {
+        tileRequestAborted = true;
+      });
       request.addEventListener('load', function load() {
-        if (!this.responseText || this.status >= 400) return;
+        if (!this.responseText || this.status >= 400 || tileRequestAborted) return;
```

Comment:
I'm not sure I understand this correctly. If the request from the current context is aborted, do we still get a `load` event? Theoretically, there should be no `load` event after calling `request.abort()`.